SPM Example/PortalSwift/ViewController.swift

```diff
@@ -210,43 +210,28 @@ class ViewController: UIViewController, UITextFieldDelegate {
     }
   }
 
-  public func eject(_ withBackupMethod: BackupMethods) async throws -> String {
-    guard let portal else {
-      self.logger.error("ViewController.eject() - ❌ Portal not initialized. Please call registerPortal().")
-      throw PortalExampleAppError.portalNotInitialized()
-    }
-    guard let user else {
-      self.logger.error("ViewController.eject() - ❌ User not logged in.")
-      throw PortalExampleAppError.userNotLoggedIn()
-    }
-    guard let config else {
-      self.logger.error("ViewController.recover() - ❌ Application configuration not set.")
-      throw PortalExampleAppError.configurationNotSet()
-    }
-    guard let cipherTextUrl = URL(
-      string: "\(config.custodianServerUrl)/mobile/\(user.exchangeUserId)/cipher-text/fetch?backupMethod=\(withBackupMethod.rawValue)"
-    ) else {
-      throw URLError(.badURL)
-    }
-    let cipherTextData = try await requests.get(cipherTextUrl)
-    let cipherTextResponse = try decoder.decode(CipherTextResult.self, from: cipherTextData)
-
-    guard let organizationBackupShareUrl = URL(
-      string: "\(config.custodianServerUrl)/mobile/\(user.exchangeUserId)/org-share/fetch?backupMethod=\(withBackupMethod.rawValue)-SECP256K1"
-    ) else {
-      throw URLError(.badURL)
+    public func eject(_ withBackupMethod: BackupMethods) async throws -> EjectedKeys {
+        guard let portal else {
+            self.logger.error("ViewController.eject() - ❌ Portal not initialized. Please call registerPortal().")
+            throw PortalExampleAppError.portalNotInitialized()
+        }
+        guard let user else {
+            self.logger.error("ViewController.eject() - ❌ User not logged in.")
+            throw PortalExampleAppError.userNotLoggedIn()
+        }
+        guard let config else {
+            self.logger.error("ViewController.recover() - ❌ Application configuration not set.")
+            throw PortalExampleAppError.configurationNotSet()
+        }
+        
+        let result = try await portal.ejectPrivateKeys(
+            withBackupMethod,
+            exchangeUserId: user.exchangeUserId,
+            custodianServerUrl: config.custodianServerUrl
+        )
+        
+        return result
     }
```

Comment:
Indentation is a bit off ^

Reply:
Yes yes, since I fromatted this func with Xcode using 'control + i', I believe the whole file needs to be formatted but I didn't want to include none-relevant changes to the PR.